### PR TITLE
Add the script presubmit-tests.sh in order to launch the prow jobs

### PR DESCRIPTION
**Pull request by houshengbo**
Partially-Closes: #7

### CODE REVIEW

File: test/presubmit-tests.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Knative Authors
```
Comment (k4leung4) [inline test/presubmit-tests.sh]: update this to 2019?

Comment (houshengbo) [reply test/presubmit-tests.sh]: MY BAD!!!

Comment (houshengbo) [reply test/presubmit-tests.sh]: done.

**Pull request merged**